roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1774,6 +1782,11 @@ def init_local_facts(self, facts=None,
                     if isinstance(val, basestring):
                         val = [x.strip() for x in val.split(',')]
                     new_local_facts['docker'][key] = list(set(val) - set(['']))
+            # Convert legacy log_options comma sep string to a list if present:
+            if 'log_options' in new_local_facts['docker'] and \
+                    isinstance(new_local_facts['docker']['log_options'], basestring) and \
+                    new_local_facts['docker']['log_options'] != '':
```

Comment:
I'm remembering an issue like this with the docker registry values as well..  maybe we need to add [""] to the things to delete below as well.